Classes/ViewHelpers/AbstractFormViewHelper.php

```diff
@@ -37,7 +35,7 @@ public function render() {
 		$container = $this->getContainer();
 		$container->add($component);
 		// rendering child nodes with Form's last sheet as active container
-		$this->viewHelperVariableContainer->addOrUpdate(self::SCOPE, self::SCOPE_VARIABLE_EXTENSIONNAME, $extensionName);
+		$this->viewHelperVariableContainer->addOrUpdate(self::SCOPE, self::SCOPE_VARIABLE_EXTENSIONNAME, NULL);
```

Comment:
I think this one should use `$this-> getExtensionName()` as default, rather than `NULL`. Most of our Form ViewHelpers will override this method anyway - but this way we will resolve the current name from VH var container if it is set.